feodo/package.yaml

```diff
@@ -21,23 +21,18 @@ pipelines:
     name: Update Feodo Context
     description: A pipeline that periodically refreshes the Feodo lookup-table context.
     definition: |
-      every 1h
-        from https://feodotracker.abuse.ch/downloads/ipblocklist_aggressive.csv read csv --allow-comments
-        | context update feodo --key=dst_ip --clear
+      // tql2
+      every 1h {
+        load_http "https://feodotracker.abuse.ch/downloads/ipblocklist_aggressive.csv"
+        read_csv allow_comments=true
+      }
+      context::update "feodo", key="dst_ip"
```

Comment:
@dominiklohmann is this correct?

Reply:
Yes. I looked at the code for this after our last discussion on this, and `--clear` was actually a no-op for quite some time now.